<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="LayerCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="legendCheck">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Legend</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QgsFieldComboBox" name="Field2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsFieldComboBox" name="Field1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LayerCombo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>Field1</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LayerCombo</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>Field2</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
